--- /Users/forrest/Code/american-software-company/scratchpad/qa-ruben/AS-70/mut/pristine/apps/invoicing/views/connect-stripe.ejs	2026-09-12 01:29:08
+++ /Users/forrest/Code/american-software-company/scratchpad/qa-ruben/AS-70/mut/f11/apps/invoicing/views/connect-stripe.ejs	2026-09-12 01:31:31
@@ -75,11 +75,6 @@
             <p><%= banner.message %></p>
           </div>
         <% } %>
-        <% if (banner.tone === 'warning') { %>
-          <div class="banner banner-warning">
-            <p><%= banner.message %></p>
-          </div>
-        <% } %>
         <% if (banner.tone === 'error') { %>
           <div class="banner banner-error">
             <p><%= banner.message %></p>
